pkgs/tools/system/sleuthkit/default.nix

```diff
@@ -17,17 +55,34 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  nativeBuildInputs = [ autoreconfHook ];
+  nativeBuildInputs = [ autoreconfHook openjdk perl ant rdeps ];
   buildInputs = [ libewf afflib openssl zlib ];
 
-  # Hack to fix the RPATH.
+  # Hack to fix the RPATH
   preFixup = "rm -rf */.libs";
 
-  meta = {
+  IVY_HOME = "/build/.ant";
+  JAVA_HOME = "${openjdk}";
```

Comment:
```suggestion
  JAVA_HOME = openjdk;
```

Reply:
I set JAVA_HOME in preUnpack too, so I kept the ${openjdk}

Reply:
Have you pushed the latest version? Currently GitHub only shows me one mention of JAVA_HOME in the code.